README.md

```diff
@@ -5,20 +5,160 @@
+
+To use `pytest-ansible-units`, follow these steps:
+
+1. Install the plugin using pip:
+
+```
+pip install pytest-ansible
+```
+
+2. Ensure you have Python 3.8 or greater, ansible-core, and pyyaml installed.
+
+3. Depending on your preferred directory structure, you can clone collections into the appropriate paths.
+
+   - **Collection Tree Approach**: The preferred approach is to clone the collections being developed into it's proper collection tree path. This eliminates the need for any
+     symlinks and other collections being developed can be cloned into the same tree structure.
+
+     ```
+     git clone <repo> collections/ansible_collections/<namespace>/<name>
+     ```
+
+     Note: Run `pytest` in the root of the collection directory, adjacent to the collection's `galaxy.yml` file.
+
+   - **Shallow Tree Approach**:
+
+     ```
+     git clone <repo>
+     ```
+
+     Notes:
+
+     - Run `pytest` in the root of the collection directory, adjacent to the collection's `galaxy.yml` file.
+     - A collections directory will be created in the repository directory, and collection content will be linked into it.
+
+4. Execute the unit tests using pytest: `pytest tests`
+
+### Help
+
+The following may be added to the collections' `pyproject.toml` file to limit warnings and set the default path for the collection's tests
+
+```
+[tool.pytest.ini_options]
+testpaths = [
+    "tests",
+]
+filterwarnings = [
+    'ignore:AnsibleCollectionFinder has already been configured',
+]
+```
+
+Information from the `galaxy.yml` file is used to build the `collections` directory structure and link the contents. The `galaxy.yml` file should reflect the correct collection namespace and name.
+
+One way to detect issues without running the tests is to run:
+
+```
+pytest --collect-only
+```
+
+The follow errors may be seen:
+
+```
+E   ModuleNotFoundError: No module named 'ansible_collections'
+```
+
+- Check the `galaxy.yml` file for an accurate namespace and name
+- Ensure `pytest` is being run from the collection's root directory, adjacent to the `galaxy.yml`
+
+```
+HINT: remove __pycache__ / .pyc files and/or use a unique basename for your test file modules
+```
+
+## Molecule Scenario Integration
+
+The `pytest-molecule` plugin enables pytest discovery of all `molecule.yml` files inside the codebase and runs them as pytest tests. It allows you to include Molecule scenarios as part of your pytest test suite, allowing you to thoroughly test your Ansible roles and playbooks across different scenarios and environments. Once you install `pytest-ansible`, you should be able to just run `pytest` to execute Molecule on all roles and scenarios.
```

Comment:
Most of this information should be moved to the "legacy" section of molecule.  The "new" approach of adding the pytest file doesn't have the marker support.

Reply:
#161